app/helpers/application_helper/dialogs.rb

```diff
@@ -88,14 +88,15 @@ def time_tag_options(field, url, hour_or_min, auto_refresh_options_hash)
 
   def drop_down_options(field, url)
     tag_options = {:class => "dynamic-drop-down-#{field.id} selectpicker"}
+    multiple = field.force_multi_value ? true : false
     extra_options = {
       "data-miq_sparkle_on"  => true,
       "data-miq_sparkle_off" => true,
-      "data-live-search"     => true
+      "data-live-search"     => true,
       # data-miq_observe functionality is handled by dialogFieldRefresh.initializeDialogSelectPicker here
     }
-
-    add_options_unless_read_only(extra_options, tag_options, field)
+    options_hash = multiple ? {:multiple => "multiple"} : {}
```

Comment:
I think I'd rather just see this as a one line if:
`extra_options.merge!(:multiple => "multiple") if multiple`